src/pages/management/configuration-api/index.mdx

```diff
@@ -2495,7 +2495,7 @@ For `bot` webhooks, you need to [create a bot](#create-bot), [register webhooks]
 | `action`<sup>**2**</sup>                        | yes      | `string`   | The [action](#triggering-actions) that triggers sending a webhook.                                                                                                                                                                                |
 | `secret_key`                                    | yes      | `string`   | The secret key sent in webhooks to verify the source of a webhook.                                                                                                                                                                                |
 | `url`                                           | yes      | `string`   | Destination URL for the webhook                                                                                                                                                                                                                   |
-| `additional_data`                               | no       | `[]string` | Additional data arriving with the webhook. Value: `chat properties`, `chat_presence_user_ids` (all actions except for `agent_status_changed`, `agent_deleted`, `incoming_customer`)                                                               |
+| `additional_data`                               | no       | `[]string` | Additional data arriving with the webhook. Value: `chat_properties`, `chat_presence_user_ids` (all actions except for `agent_status_changed`, `agent_deleted`, `incoming_customer`)                                                               |
```

Comment:
here and below, I just added one case from `platform-docs` as it is a tiny typo in all the versions but may cause problems for the users.